### PR TITLE
(maint) Fix rubocop offenses

### PR DESCRIPTION
**Pull request by glennsarti**
This commit updates VM Pooler to fix any existing rubocop offenses and also
fixes any offenses in the lib/vmpooler.rb file.  This commit also regenerates
the Todo file.

### CODE REVIEW

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -1,526 +1,505 @@
...
+    - 'lib/vmpooler/api/v1.rb'
+    - 'lib/vmpooler/pool_manager.rb'
+    - 'lib/vmpooler/vsphere_helper.rb'
+
+# Offense count: 2
+# Cop supports --auto-correct.
+# Configuration parameters: SupportedStyles, WordRegex.
+# SupportedStyles: percent, brackets
+Style/WordArray:
+  EnforcedStyle: percent
```
Comment (shrug) [inline .rubocop_todo.yml]: Stray carriage return?

File: .rubocop_todo.yml [hunk 2]
```diff
@@ -1,526 +1,505 @@
...
+  MinSize: -Infinity
```
Comment (shrug) [inline .rubocop_todo.yml]: Stray carriage return?

File: lib/vmpooler/pool_manager.rb [hunk 3]
```diff
@@ -436,7 +436,7 @@ def check_disk_queue(maxloop = 0, loop_delay = 5)
 
           unless maxloop.zero?
             break if loop_count >= maxloop
-            loop_count = loop_count + 1
+            loop_count += 1
```
Comment (shrug) [inline lib/vmpooler/pool_manager.rb]: 💯

File: .rubocop.yml [hunk 4]
```diff
@@ -29,13 +29,15 @@ Metrics/PerceivedComplexity:
   Enabled: false
 Metrics/CyclomaticComplexity:
   Enabled: false
-#   - Long Methods, Classes and blocks
+#   - Long Methods, Classes, Blocks and Modules
```
Comment (mwaggett) [inline .rubocop.yml]: Oxford comma? 😄

File: .rubocop_todo.yml (hunk 2 above)
Comment (glennsarti) [inline .rubocop_todo.yml]: Hrmm....That's autogenerated from rubocop.  Will fix manually.

File: .rubocop.yml (hunk 4 above)
Comment (glennsarti) [inline .rubocop.yml]: Seriously? :-)

File: .rubocop_todo.yml (hunk 2 above)
Comment (glennsarti) [inline .rubocop_todo.yml]: I wonder if that's rubocop trying to be useful because I'm using Windows.

Comment (glennsarti) [inline .rubocop_todo.yml]: Well...looks like the autogeneration on Windows creates a CRLF file which git _mostly_ converts to LF for me when on Windows.  WIll have to keep an eye on that.

File: .rubocop.yml (hunk 4 above)
Comment (glennsarti) [inline .rubocop.yml]: Updated

**Pull request merged**